pydantic_ai_slim/pydantic_ai/models/gemini.py

```diff
@@ -91,7 +91,7 @@ class GeminiModel(Model):
     _provider: Literal['google-gla', 'google-vertex'] | Provider[AsyncHTTPClient] | None = field(repr=False)
     _auth: AuthProtocol | None = field(repr=False)
     _url: str | None = field(repr=False)
-    _system: str | None = field(default='google-gla', repr=False)
+    _system: str = field(default='gemini', repr=False)
```

Comment:
is this what we are supposed to use?

Reply:
https://github.com/pydantic/pydantic-ai/pull/1121#discussion_r1995599112